apps/backend/src/modules/project-api-keys/project-api-key.controller.ts

```diff
@@ -69,6 +69,17 @@ class ProjectApiKeyController extends BaseController {
 				body: projectApiKeyCreateValidationSchema,
 			},
 		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.delete(
+					options as APIHandlerOptions<{
+						params: { projectId: string };
+					}>,
+				),
+			method: "DELETE",
+			path: ProjectApiKeysApiPath.$ID,
+		});
```

Comment:
We don't need a route for deletion either, this should happen on the service level in the create method because it's a part of a creation logic:
- if we already have API key for project - then remove it
- then create a new API key
From the frontend perspective, everything is the same, except that we no longer block the button.

Reply:
This looks like a much simpler solution than the one I have been using